core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -6069,6 +6069,21 @@ private void checkLiteral2(String expression, String expected) {
     sql(query).withLibrary(SqlLibrary.POSTGRESQL).ok(expected);
   }
 
+  /** Test case for
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-7088">[CALCITE-7088]
+   * X LIKE '%%' should simply to X=X</a>. */
```

Comment:
RelToSqllComverterTest tests dialect support. This feature is about simplification. So this test should not be here. 

Nowhere do you test what the summary says - that ‘x like “%%”’ simplifies to ‘x = x’. Is that even true?